tildeplays.py

```diff
@@ -1,8 +1,7 @@
 from datetime import datetime
 from os import environ, path
 from textwrap import wrap
-import time
-import subprocess
+import time,subprocess,launch,traceback
```

Comment:
Stylistically, I prefer each import to be on its own line for readability purposes. The exception being `from x import y`.